executor/tests/test_config.py

```diff
@@ -57,17 +60,77 @@ def test_initialize_config_defaults(self):
     def test_initialize_config_custom(self):
         environment = {'EXECUTOR_MAX_BYTES_READ_PER_LINE': '1234',
                        'EXECUTOR_MAX_MESSAGE_LENGTH': '1024',
+                       'EXECUTOR_PROGRESS_OUTPUT_FILE': 'progress_file',
                        'MESOS_EXECUTOR_SHUTDOWN_GRACE_PERIOD': '4secs',
                        'MESOS_SANDBOX': '/sandbox/location',
-                       'PROGRESS_OUTPUT_FILE': 'progress_file',
                        'PROGRESS_REGEX_STRING': 'progress/regex',
                        'PROGRESS_SAMPLE_INTERVAL_MS': '2500'}
         config = cc.initialize_config(environment)
 
         self.assertEqual(1234, config.max_bytes_read_per_line)
         self.assertEqual(1024, config.max_message_length)
+        self.assertEqual('EXECUTOR_PROGRESS_OUTPUT_FILE', config.progress_output_env_variable)
         self.assertEqual('progress_file', config.progress_output_name)
         self.assertEqual('progress/regex', config.progress_regex_string)
         self.assertEqual(2500, config.progress_sample_interval_ms)
         self.assertEqual('/sandbox/location', config.sandbox_directory)
         self.assertEqual(4000, config.shutdown_grace_period_ms)
+
+    def test_initialize_config_custom_progress_file_without_sandbox(self):
+        environment = {'EXECUTOR_MAX_BYTES_READ_PER_LINE': '1234',
+                       'EXECUTOR_MAX_MESSAGE_LENGTH': '1024',
+                       'MESOS_EXECUTOR_SHUTDOWN_GRACE_PERIOD': '4secs',
+                       'OUTPUT_TARGET_FILE': 'progress.out',
+                       'EXECUTOR_PROGRESS_OUTPUT_FILE_ENV': 'OUTPUT_TARGET_FILE',
```

Comment:
For me, it's more clear to have the `_ENV` entry listed first, since it is kind of like a "pointer" to the actual file entry (occurs in multiple places)

Reply:
Sorted properly, which caused it to move above.